go/common/grpc/testing/ping.go

```diff
@@ -0,0 +1,207 @@
+// Package testing implements common grpc testing helpers.
+package testing
+
+import (
+	"context"
+	"time"
+
+	"github.com/oasislabs/oasis-core/go/common/pubsub"
+	"google.golang.org/grpc"
+)
+
+var (
+	_ PingServer = (*pingServer)(nil)
+	_ PingClient = (*pingClient)(nil)
+)
+
+// PingQuery is the PingServer query.
+type PingQuery struct {
+}
+
+// PingResponse is the response of the PingServer.
+type PingResponse struct {
+}
+
+// PingServer is a testing ping server interface.
+type PingServer interface {
```

Comment:
Hm what's the relationship between this server/service/etc. and the one defined in the policy test? Could these be unified?

Reply:
done